.github/workflows/common-app-build.yml

```diff
@@ -0,0 +1,90 @@
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      docker-repo:
+        required: true
+        type: string
+      cache-version:
+        required: false
+        type: string
+        default: v1
+      is-package:
+        required: true
+        type: string
+      renv-version:
+        required: false
+        type: string
+        default: 0.14.0
+        
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: powerex/pwx-r:4.1.2
+    env:
+      GITHUB_PAT: "${{ secrets.ACCESS_TOKEN_GITHUB }}"
+      working-directory: ${{ inputs.working-directory }}
+      docker-repo: ${{ inputs.docker-repo }}
```

Comment:
tu to nemusis redefinovat ako env premenne - rovno aj nizsie mozes vsade robit `${{ inputs.working-directory }}` a `${{ inputs.docker-repo }}`

Reply:
ok upravil som to